spec/functional/specific_pages_titles_spec.rb

```diff
@@ -15,21 +17,11 @@
     test.webdriver.quit
   end
 
-  it 'DesktopEditors Install flatpak title' do
-    page_to_open = "#{test.webdriver.get_url}installation/desktop-install-flatpak.aspx"
-    test.webdriver.open(page_to_open)
-    expect(test.webdriver.get_title_of_current_tab).to eq('How to install ONLYOFFICE Desktop Editors for Linux to your computer using Flatpak? - ONLYOFFICE')
-  end
-
-  it 'DesktopEditors Install Ubuntu title' do
-    page_to_open = "#{test.webdriver.get_url}installation/desktop-install-ubuntu.aspx"
-    test.webdriver.open(page_to_open)
-    expect(test.webdriver.get_title_of_current_tab).to eq('Installing ONLYOFFICE Desktop Editors to Debian, Ubuntu and derivatives - ONLYOFFICE')
-  end
-
-  it 'DesktopEditors Install RHEL title' do
-    page_to_open = "#{test.webdriver.get_url}installation/desktop-install-rhel.aspx"
-    test.webdriver.open(page_to_open)
-    expect(test.webdriver.get_title_of_current_tab).to eq('Installing ONLYOFFICE Desktop Editors to Red Hat, CentOS and derivatives - ONLYOFFICE')
+  page_hash.hash.each do |hash|
+    it "check page title of #{hash[:url]}" do
```

Comment:
I think some quotes around url allow better readability in results, and make first symbol capital

Like:
```
it "Check page title of '#{hash[:url]}'" do
```